src/components/tooltipMdPanel/tooltip.js

```diff
@@ -0,0 +1,346 @@
+/**
+ * @ngdoc module
+ * @name material.components.tooltipMdPanel
+ */
+angular
+  .module('material.components.tooltipMdPanel', [ 'material.core' ])
+  .directive('mdTooltipMdPanel', MdTooltipDirective);
```

Comment:
no. please do not do this; we should not have an `tooltipMdPanel` component.
Please just use a `wip/panel_tooltip` branch and then modify the tooltip.js.